components/support/migration/src/main/java/mozilla/components/support/migration/MigrationResultsStore.kt

```diff
@@ -56,6 +56,7 @@ internal class MigrationResultsStore(context: Context) : SharedPreferencesCache<
                 Migration.History.javaClass.simpleName -> Migration.History
                 Migration.Bookmarks.javaClass.simpleName -> Migration.Bookmarks
                 Migration.OpenTabs.javaClass.simpleName -> Migration.OpenTabs
+                Migration.Gecko.javaClass.simpleName -> Migration.Gecko // Bzzzp!
```

Comment:
Great comment ;)

Reply:
Heh, yeah, this caused a crash at runtime when missing and the compiler didn't notify me about that since it doesn't know how to make this when statement include all cases (like for enums or sealed classes). Also using `simpleName` is a bit scary here since that can easily change and then breaks at runtime when reading an older state from disk.